lxd/api_cluster.go

```diff
@@ -4479,9 +4479,25 @@ func evacuateClusterSelectTarget(ctx context.Context, s *state.State, inst insta
 			return fmt.Errorf("Failed getting cluster members: %w", err)
 		}
 
+		// Filter candidates by group if needed.
+		_, group := shared.TargetDetect(inst.LocalConfig()["volatile.cluster.target"])
```

Comment:
It looks like `TargetDetect` is only used by LXD, so this would be a good opportunity to move it out of `shared` and update it to use the targetGroupPrefix constant too.